src/components/Header/TopBar.tsx

```diff
@@ -23,13 +25,14 @@ const StyledLink = styled(ExternalLink)`
 
 const TopBar = () => {
   const ethPrices = useEthPrices()
+  const [activeNetwork] = useActiveNetworkVersion()
   return (
     <Wrapper>
       <RowBetween>
         <Polling />
         <AutoRow gap="6px">
           <RowFixed>
-            <Item>ETH Price:</Item>
+            {activeNetwork.id === SupportedNetwork.CELO ? <Item>Celo Price:</Item> : <Item>Eth Price:</Item>}
```

Comment:
I'd prefer this be expressed differently so that future chains have an easier time updating it.

    let nativeAsset: string
    switch (activeNetwork.id) {
      case SupportedNetwork.Celo:
        nativeAsset = 'Celo'
        break
      default:
        nativeAsset = 'Eth'
        
    ...
    <Item>{nativeAsset} Price:</Item>

Reply:
Yeah this is solid 👌

Reply:
(will create a followup PR)